lms/djangoapps/shoppingcart/migrations/0004_change_meta_options.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-05-14 20:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shoppingcart', '0003_auto_20151217_0958'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='invoiceitem',
+            options={'base_manager_name': 'objects'},
```

Comment:
Didn't get a chance to ask on the other PR, but I wasn't sure how this change was related to the work. Not blocking, just curious.

Reply:
It was merely to remove a deprecation warning. I'm attempting to stop ignoring them and fix them - at least the easy ones...